cirq/sim/wave_function_test.py

```diff
@@ -25,6 +25,102 @@ def assert_dirac_notation(vec, expected, decimals=2):
     assert cirq.dirac_notation(np.array(vec), decimals=decimals) == expected
 
 
+def test_bloch_vector_simple():
+    sqrt = np.sqrt(0.5)
+    test_state = np.array([sqrt, sqrt])
+
+    bloch = cirq.bloch_vector_from_state_vector(test_state, 0)
```

Comment:
Test that the output is correct for the |0>+|1> state, the |0>+i|1> state, the |T> state, and the state that should produce the bloch vector x=y=z=sqrt(1/3).